teku/src/main/java/tech/pegasys/teku/cli/options/ValidatorOptions.java

```diff
@@ -20,12 +20,25 @@
 import tech.pegasys.teku.util.cli.GraffitiConverter;
 
 public class ValidatorOptions {
+  @Option(
+      names = {"--validator-keys"},
+      paramLabel = "<KEY_PATH>:<PASS_PATH>",
+      description =
+          "Where KEY_PATH is either a directory or file, and PASS_PATH is the associated password path. "
+              + "keysDir:passDir will find keysDir/**.json, and expect to find passDir/**.txt. "
+              + "keyFile:passFile will expect that the file 'keyFile' exists, "
+              + "and the file containing the password for it is 'passFile'. "
+              + "The path separator is operating system dependent, and should be ';' in windows rather than ':'.",
```

Comment:
@bgravenorst wondering if I could get some help explaining this? It's currently pretty long, and you might have ideas about how i can clean it up